R/expr-string.R

```diff
@@ -407,16 +410,10 @@ expr_str_strip_chars_end <- function(matches = NULL) {
 #'
 #' @inherit as_polars_expr return
 #' @examples
-#' some_floats_expr <- pl$lit(c(0, 10, -5, 5))
-#'
-#' # cast to String and ljust alignment = 5, and view as R char vector
-#' some_floats_expr$cast(pl$String)$str$zfill(5)$to_r()
-#'
-#' # cast to int and the to utf8 and then ljust alignment = 5, and view as R
-#' # char vector
-#' some_floats_expr$cast(pl$Int64)$cast(pl$String)$str$zfill(5)$to_r()
-expr_str_zfill <- function(alignment) {
-  self$`_rexpr`$str_zfill(as_polars_expr(alignment)$`_rexpr`) |>
+#' df <- pl$DataFrame(a = c(-1L, 123L, 999999L, NA))
+#' df$with_columns(zfill = pl$col("a")$cast(pl$String)$str$zfill(4))
```

Comment:
Old example was outdated in terms of functions and unclear. The new example is the one in py-polars